.storybook/main.js

```diff
@@ -1,30 +1,28 @@
-const path = require("path");
-const { resolve } = require("path");
+const path = require("path")
 const bloomTheme = require("../tailwind.config.js");
 const tailwindVars = require("../tailwind.tosass.js")(bloomTheme);
-const { NormalModuleReplacementPlugin } = require("webpack");
 
 module.exports = {
-  stories: ["../src/**/*.stories.@(tsx|mdx)"],
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
-    "@storybook/addon-actions",
-    "@storybook/addon-docs",
-    "@storybook/addon-a11y",
```

Comment:
Curious why some of these addons were removed?

Reply:
I ran into some issues while upgrading and kept removing things until it worked. I forgot to re-add these addons. I just re-added them and they appear to work! Thanks for catching this